src/declarative-stack.ts

```diff
@@ -20,15 +20,22 @@ export class DeclarativeStack extends cdk.Stack {
   constructor(scope: cdk.App, id: string, props: DeclarativeStackProps) {
     super(scope, id, props);
 
-    const ctx = props.template.metadata.get('AWS::CDK::Context') ?? {};
+    const {
+      template: { description, metadata, templateFormatVersion },
```

Comment:
I hadn't realised you could destructure like this. It makes sense in hindsight :)